app/src/main/java/com/kickstarter/viewmodels/CommentsViewHolderViewModel.kt

```diff
@@ -22,6 +22,9 @@ interface CommentsViewHolderViewModel {
         /** Call when the user clicks reply to comment */
         fun onReplyButtonClicked()
 
+        /** Call when the user clicks view replies to comment */
+        fun onViewRepliesButtonClicked()
```

Comment:
There should be tests covering the new inputs/outputs

Reply:
done